docs/source/pillars/computing_technology.md

```diff
@@ -100,6 +135,12 @@ _The ability to run analyses requiring more compute resources, or more specialis
 * - Statement
   - Guidance
   - Importance
+* - Your TRE must maintain segregation of users and data from different projects when using non-standard compute.
+  - High performance or specialist compute is often shared amongst multiple users.
+    Users and data must remain segregated at all times.
+    For example, when using physical compute resources, all sensitive data could be securely wiped before another user is given access to that same node.
```

Comment:
Is wording on could be strong enough here, if we are saying it's mandatory

Reply:
- Keep users and data separated (this is mandatory)
- All sensitive data could be securely wiped (one possible way of achieving the above)